src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,116 @@
+package core.basesyntax.service;
+
+import static core.basesyntax.db.Storage.getPeople;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private StorageDao storageDao = new StorageDaoImpl();
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+    private Storage storage = new Storage();
+    private User bob;
+
+    @BeforeEach
+    void setUp() {
+        bob = new User("bob423", "123456", 19);
+    }
+
+    @AfterEach
+    void tearDown() {
+        new Storage();
+    }
+
+    @Test
+    void register_registerUsersByCorrectData_ok() {
```

Comment:
You are not calling the register method, and we need to check that this logic works correctly.